example/App.vue

```diff
@@ -332,73 +332,73 @@
     <input name="show" type="checkbox" :checked="showAll" @change="extend" />
     <label for="show">Show All</label>
     <section v-if="showAll">
-      <div class="row" v-for="icon in icons" :key="icon.options.name">
-        <span class="name">{{ icon.options.name }}</span>
-        <div class="icons" :title="icon.options.name">
-          <component :is="icon.options.name" />
+      <div class="row" v-for="icon in icons" :key="icon">
```

Comment:
Do components get properly stringified in this case? If so, neat.

Reply:
I changed the assignment to `icons` to an `Object.keys` call, so it’s just the string names of the components. No magic here.

The demo app now globally loads the plug-in the same way a consumer app would, using `app.use(PhosphorVue)` in the entry point, so there wasn’t a need to dynamically register all the component objects anymore and I just dropped the references to them.

Reply:
Gotcha, missed that the first time around! Does this give us tree-shaking as well, if you were to only `app.use(SomeIcon)`?

Reply:
You couldn't `app.use(SomeIcon)`, because the icons aren't `Plugin` typed (i.e. they don't have an `install` function), but you should be able to import individual icon components like `import { PhFoo, PhBar, PhBaz } from 'phosphor-vue'` and that should theoretically get tree-shaken.

You could do it in the app entry point to globally register them like `app.component(PhFoo)` if you expect to use it a lot of places, or you could import it in a specific SFC and locally register it there.

Reply:
^ that is assuming I configure Rollup properly to build anything at all 😆